ccx_messaging/error.py

```diff
@@ -25,8 +25,4 @@ class CCXMessagingError(Exception):
 
     def format(self, input_msg):
         """Format the error by adding information about input Kafka message."""
-        return (
-            f"Status: Error; "
-            f"Topic: {input_msg['topic']}; "
-            f"Cause: {self}"
-        )
+        return f"Status: Error; " f"Topic: {input_msg['topic']}; " f"Cause: {self}"
```

Comment:
you can use just one f-string

Reply:
Yes, this was auto-formatted by `black`. I'll fix it